common/views/components/RenderlessPaginator/RenderlessPaginator.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+import { Fragment, type Node as ReactNode } from 'react';
+
+type Link = {|
+  +pathname: string,
+  +query: Object,
+|};
+
+type LinkProps = {|
```

Comment:
[There's a type for this](https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/model/next-link-type.js)

Reply:
Oh yeah? Think this was copy/pasta from `Paginator`